tests/common/experimental/test_reducers_and_aggregators.py

```diff
@@ -48,25 +49,35 @@
 default_test_quantile = 0.1
 
 
+REF_TYPE = List[Union[float, "REF_TYPE"]]
+
+
+@dataclass
+class OfflineAggregatorTestCase:
+    aggregation_axes: Optional[Tuple[int, ...]]
```

Comment:
`Optional[AggregationAxes]`

Reply:
I suggest to do this in your PR as it is already AggregationAxes there, what do you think?

Reply:
Same - I didn't do modifications to aggregation axes there. I checked once again, and in fact, you are only introducing the aggregation axes in this PR, so it makes sense to set up the `AggregationAxes` type now, while all the uses of the new entity are still easily visible in the diff.

Reply:
Sorry for misleading, done